wp-content/plugins/hurumap/src/FlourishBlock/Save.js

```diff
@@ -8,6 +8,13 @@ function Save({ attributes }) {
       data-chart-id={attributes.chartId}
       data-chart-title={attributes.title}
       data-chart-description={attributes.description}
+      data-show-insight={attributes.showInsight}
+      data-insight-title={attributes.insightTitle}
+      data-insight-summary={attributes.insightSummary}
+      data-data-link-title={attributes.dataLinkTitle}
+      data-analysis-country={attributes.analysisCountry}
```

Comment:
Since we've already create the link, can't this just be `data-analysis-link-href` or `data-analysis-link-url` instead `data-analysis-country`? The props should match what we defined in HURUmap UI.

Reply:
:+1: